src/functions/types.ts

```diff
@@ -349,6 +350,12 @@ export type SlackFunctionType<Definition> = Definition extends
           FunctionDefinitionArgs<I, O, RI, RO>
         >,
       ): SlackFunctionType<Definition>;
+      addBlockSuggestionHandler(
```

Comment:
I know, if we add this change, we should consistently update other handler definitions too but what do you think about something like this? (disclaimer: the diff is incomplete - just for demo purpose)

```diff
diff --git a/src/functions/interactivity/types.ts b/src/functions/interactivity/types.ts
index 0c9bd30..6d17983 100644
--- a/src/functions/interactivity/types.ts
+++ b/src/functions/interactivity/types.ts
@@ -28,8 +28,9 @@ export type BlockSuggestionHandler<Definition> = Definition extends
   FunctionDefinitionArgs<infer I, infer O, infer RI, infer RO> ? {
     (
       context: SuggestionContext<FunctionRuntimeParameters<I, RI>>,
-      // deno-lint-ignore no-explicit-any
-    ): Promise<any> | any;
+    ): Promise<{ options: [] } | { option_groups: [] }> | { options: [] } | {
+      option_groups: [];
+    };
   }
   : never;
 
diff --git a/src/functions/slack-function_test.ts b/src/functions/slack-function_test.ts
index fd81222..4cf0051 100644
--- a/src/functions/slack-function_test.ts
+++ b/src/functions/slack-function_test.ts
@@ -110,7 +110,11 @@ Deno.test("addBlockSuggestionHandler", async () => {
 
   const actionId = "whatever";
   const suggestionSpy = mock.spy();
-  typedHandlers.addBlockSuggestionHandler(actionId, suggestionSpy);
+  typedHandlers.addBlockSuggestionHandler(actionId, async ({ inputs }) => {
+    console.log(inputs);
+    // return await { options: [] };
+    return await { option_groups: [] };
+  });
 
   await typedHandlers.blockSuggestion({ body: { action_id: actionId } });
   mock.assertSpyCalls(suggestionSpy, 1);
```

Reply:
@seratch I made another pass at adding block-suggestion handler-specific return types. I think it is better now. Can you tell me what you think?

Reply:
@filmaj I don't remember both plain_text and mrkdwn formats are supported for text, label but apart from that, it looks great!

Reply:
According to [the reference docs (under Fields)](https://api.slack.com/reference/block-kit/composition-objects#option), only plain_text is supported for select / dropdown / overflow menus:

> A [text object](https://api.slack.com/reference/block-kit/composition-objects#text) that defines the text shown in the option on the menu. Overflow, select, and multi-select menus can only use plain_text objects, while radio buttons and checkboxes can use mrkdwn text objects. Maximum length for the text in this field is 75 characters.